Add info about .desktop file in linux quickstart (#7)
wavetermdev/waveterm-docs: quickstart.mdx

    1. [Download Wave](https://www.commandline.dev/download) and extract the file
    2. Change to the Wave directory ```cd waveterm-linux-* ```
    3. Run the application:  ```./Wave```
    

>     While we work on improving the packaging for linux, you can optionally create a ```.desktop``` file , allowing you to launch Wave directly.

Reviewer: nit: 
```suggestion
    While we work on improving the packaging for Linux, you can optionally create a `.desktop` file, allowing you to launch Wave directly.
```

>     After doing the above steps, create a file named `Wave.desktop`, and add the following, modifying the `Exec` path to point to your Wave installation:

Reviewer: ```suggestion
    After doing the above steps, create a file named `Wave.desktop`, and add the following, modifying the `Exec` and `Icon` paths to point to your Wave installation:
```

    ```
    [Desktop Entry]
    Type=Application
    Name=Wave

>     Icon= # IMP todo after PR discussion

Reviewer: ```suggestion
    Icon= /path/to/Wave-linux-*/resources/app/public/waveterm.icns
```

>     Exec=/path/to/Wave-linux-x64/Wave

Reviewer: ```suggestion
    Exec=/path/to/Wave-linux-*/Wave
```

    # setting this to true will launch an extra terminal to run the Wave command
    Terminal=false
    ```

>     Then run ```desktop-file-validate Wave.desktop``` which should validate the file. Then, run ```sudo cp Wave.desktop ~/.local/share/applications/Wave.desktop```.

Reviewer: ```suggestion
    Run `desktop-file-validate Wave.desktop` to validate that the desktop entry is correct, then, run `sudo cp Wave.desktop ~/.local/share/applications/Wave.desktop` to ensure the entry shows up with your other desktop applications.
```
